docs/project/evolution.md

```diff
@@ -42,7 +42,7 @@ This process is designed to:
 
 -   Ensure these kinds of changes can receive feedback from the entire
     community.
--   Resolve questions and decide direction efficiently.
+-   Resolve questions and decide directions efficiently.
```

Comment:
direction here should remain singular -- i.e., we're taking Carbon on one route, not multiple.